bokeh/plotting/helpers.py

```diff
@@ -821,31 +862,34 @@ def func(self, **kwargs):
             kwargs['global_alpha'] = kwargs['alpha']
 
         # handle the main glyph, need to process literals
-        glyph_ca = _pop_colors_and_alpha(glyphclass, kwargs)
+        glyph_ca = _pop_visuals(glyphclass, kwargs)
         incompatible_literal_spec_values = []
         incompatible_literal_spec_values += _process_sequence_literals(glyphclass, kwargs, source, is_user_source)
         incompatible_literal_spec_values += _process_sequence_literals(glyphclass, glyph_ca, source, is_user_source)
         if incompatible_literal_spec_values:
             raise RuntimeError(_GLYPH_SOURCE_MSG % nice_join(incompatible_literal_spec_values, conjuction="and"))
 
+        # use glyph_ca as default for other glyphs
+        default_ca = glyph_ca
```

Comment:
It is logically innocuous, but it made it locally obvious what is happening in case someone doesn't know the arguments of `_pop_visuals`. It can be taken down, no prob.

Reply:
Actually, I probably set that `default_ca` before choosing/noting that the keyword argument is explicit. Since we're always called `_pop_visuals( ..., defaults=glyph_ca, ...)` this is really redundant. Gone!